brianchandotcom/liferay-portal
# LPS-141230 Test that verifies a Custom Element can be displayed in a portlet

Forwarded from: https://github.com/liferay-frontend/liferay-portal/pull/1707 (Took 1 `ci:forward` attempt in 6 minutes)
[Console](https://test-1-28.liferay.com/job/test-portal-source-format/992//consoleFull)

@sergiojimcos
@liferay-frontend

Original pull request comment:
JIRA Ticket: https://issues.liferay.com/browse/LPS-141230

**Test Scenario**:

1. Create a Custom Element Widget referencing vanilla counter custom element
2. Custom Element Widget on content page
3.  Navigate to content page
4. See the Vanilla Counter is displayed





<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b7534286b84c9791bda8bc619e0d631d07f44379">b7534286b84c9791bda8bc619e0d631d07f44379</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/sergiojimcos/liferay-portal/tree/LPS-141230">LPS-141230</a><br/>Branch GIT ID: <a href="https://github.com/sergiojimcos/liferay-portal/commit/97a0323f47947b4d1f95145326e7864696ec0d48">97a0323f47947b4d1f95145326e7864696ec0d48</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-source-format/992/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/992/jenkins-report.html">here</a>.</h5></details></html>
